app/admin/members.rb

```diff
@@ -40,10 +42,8 @@
     column :last_name
     column :phone_number
     column :role
-    column("3 heures de #{l 1.month.ago, format: '%B'}?") { |member| member.worked_three_hours?(1.month.ago) }
-    column("3 heures de #{l Date.current, format: '%B'}?") { |member| member.worked_three_hours?(Date.current) }
-    column("3 heures de #{l 1.month.from_now, format: '%B'}?") { |member| member.worked_three_hours?(1.month.from_now) }
     column(:group) { |member| member.groups.map(&:name).join(', ') }
+    column :worked_hours_of_three_last_months_csv_version
```

Comment:
Check that the column has a translated name in the generated CSV.

Reply:
done